### ISSUE TITLE
Have students got younger? 

### ISSUE THREAD

**Issue by Dunn-R** [wmcadigital/wmn-find-a-ticket#104]
![image](https://user-images.githubusercontent.com/92024188/136210810-63b9167c-17bc-440c-8210-05d43c5e16da.png)

**Comment by houbly** [comment]
@jkmnch do you know what's correct here?

**Comment by Dunn-R** [comment]
yes, i think you are right. I just wanted to note the age boxes were slightly different. The greater detail on the screens does explain this fact
